### ISSUE TITLE
[Screenshots] Argo CD under 'How it Works See It in Action' section

### ISSUE THREAD

**Issue by saurabh100ni** [layer5io/layer5#4846]
### Current Behavior
Currently, [every integration](https://layer5.io/cloud-native-management/meshery/integrations/argo-cd) has only two old screenshots of the designer and visualizer in the 'How it Works See It in Action' section.

### Desired Behavior
- Add new images and videos from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap))
- Add screenshots for [Argo CD](https://layer5.io/cloud-native-management/meshery/integrations/argo-cd) integration.

### Acceptance Tests
- Ensure your screenshot represents the use of a few specific integrations.
- The screenshot from the Designer mode should contain a valid design, refer to [meshery catalog](https://meshery.io/catalog) to use and view published designs.
- Add both Dark Mode and Light Mode screenshots.
- The screenshot should not be a snippet, take the full screenshot of your browser.
- Use Incognito mode on Chrome browser, to ensure your bookmark bar or any other items are not shown in the screenshot/video.

---

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by singh1203** [comment]
@saurabh100ni @Ghat0tkach can I go for another one?
Thank you.

**Comment by saurabh100ni** [comment]
Go On @singh1203 😎🚀

**Comment by singh1203** [comment]
> Go On @singh1203 😎🚀

Thank you.

**Comment by leecalcote** [comment]
This manifest can be imported and used for screenshot - https://raw.githubusercontent.com/argoproj/argo-cd/master/manifests/install.yaml

**Comment by singh1203** [comment]
> This manifest can be imported and used for screenshot - https://raw.githubusercontent.com/argoproj/argo-cd/master/manifests/install.yaml

Thank you @leecalcote for the manifest but I am getting this error "Failed to upload design file: invalid design-file format: missing services field". Please assist me with this.

**Comment by leecalcote** [comment]
Gotcha. Please sure to import as an application, not as a design.

**Comment by Bprathmesh** [comment]
Please assign this issue to me

**Comment by singh1203** [comment]
> Please assign this issue to me

Please confirm if you were able to set up and run the Layer5 repository locally before getting assigned to this issue. Thank you, @Bprathmesh, for raising this.
cc - @Ashparshp

**Comment by Bprathmesh** [comment]
yeah @singh1203 @narasimha-1511  i have setup can you assign me the issue ?

**Comment by narasimha-1511** [comment]
GO on @Bprathmesh 🚀
